share/wake/lib/core/list.wake

```diff
@@ -790,6 +819,17 @@ export def tab (f: Integer => a): Integer => List a =
 export def seq: Integer => List Integer =
     tab (_)
 
+# Create a list identical elements, with a given length.
+#
+# Example:
+# ```
+#   seq 5 "a" = "a", "a", "a", "a", "a", Nil
+#   seq (-1) "a" = Nil
+#   seq 0 "a" = Nil
```

Comment:
```suggestion
#   replicate 5 "a" = "a", "a", "a", "a", "a", Nil
#   replicate (-1) "a" = Nil
#   replicate 0 "a" = Nil
```

Reply:
Yeah, that would be the thing I had a feeling I was forgetting.